### ISSUE TITLE
Sprite art enemies look ridiculous

### ISSUE THREAD

**Issue by lincolnfrog** [luqui/ichor#4]
The enemies are totally goofy and clash with the cool physics-y style of the game. Please replace with something more iconic and less representational.

**Comment by luqui** [comment]
I agree they clash.  Updated them recently from programmer art, like, dumb gray sphere instead of buddha.  I think the buddha is an improvement, at least, if a bit comical... I'm not sure what to do to reduce clash.   Ideas?
